crates/ironrdp-pdu/Cargo.toml

```diff
@@ -22,6 +22,7 @@ alloc = ["ironrdp-error/alloc"]
 
 [dependencies]
 bitflags = "2"
+const_format = "0.2"
```

Comment:
In which way this new dependency is used inside `ironrdp-pdu`? I couldn’t find by "ctrl-f"ing the commit diff (I didn’t review the whole PR yet)
Since this is bringing a procedural macro, its use should be very compelling as it’s desirable to keep core crates build fast. Hopefully, it doesn’t appear to requires the `syn` crate depending on the feature flags we enable, so it might be acceptable depending on the use case. Ultimately I would like core crates to have no dependency on the `syn` crate, which is one of the most heavy dependency out there. This will allow `cargo` to build all the IronRDP core crates before or in parallel of `syn` even if it is used by some higher level crate.

Reply:
Ah... I just forgot to remove it, I initially planned to use it but abandoned that idea, thanks!